src/MapboxStyleParser.ts

```diff
@@ -0,0 +1,1087 @@
+    }
+
+    /**
+     * Creates a GeoStylerStyle-MarkSymbolizer from a Mapbox Style Layer
+     *
+     * @param {any} layer A Mapbox Style Layer
+     * @return {MarkSymbolizer} A GeoStylerStyle-MarkSymbolizer
+     */
+    getMarkSymbolizerFromMapboxLayer(paint: any): MarkSymbolizer {
+        // TODO parse MarkSymbolizer
+        return {
+            kind: 'Mark',
+            wellKnownName: 'Circle'
+        };
+    }
+
+    /**
+     * Creates a GeoStylerStyle-IconSymbolizer from a Mapbox Style Layer
+     *
+     * @param {any} layer A Mapbox Style Layer
+     * @return {IconSymbolizer} A GeoStylerStyle-IconSymbolizer
+     */
+    getIconSymbolizerFromMapboxLayer(paint: any): IconSymbolizer {
+        // TODO parse IconSymbolizer
+        return {
+            kind: 'Icon'
+        };
+    }
+
+    /**
+     * Creates a GeoStylerStyle-TextSymbolizer from a Mapbox Style Layer
+     *
+     * @param {any} layer A Mapbox Style Layer
+     * @return {TextSymbolizer} A GeoStylerStyle-TextSymbolizer
+     */
+    getTextSymbolizerFromMapboxLayer(paint: any): TextSymbolizer {
+        return {
+            kind: 'Text',
+            spacing: paint['symbol-spacing'],
+            avoidEdges: paint['symbol-avoid-edges'],
+            pitchAlignment: paint['text-pitch-alignment'],
+            rotationAlignment: paint['text-rotation-alignment'],
+            label: this.getLabelFromTextField(paint['text-field']),
+            font: paint['text-font'],
+            size: paint['text-size'],
+            maxWidth: paint['text-max-width'],
+            lineHeight: paint['text-line-height'],
+            letterSpacing: paint['text-letter-spacing'],
+            justify: paint['text-justify'],
+            anchor: paint['text-anchor'],
+            maxAngle: paint['text-max-angle'],
+            rotate: paint['text-rotate'],
+            padding: paint['text-padding'],
+            keepUpright: paint['text-keep-upright'],
+            transform: paint['text-transform'],
+            offset: paint['text-offset'],
+            allowOverlap: paint['text-allow-overlap'],
+            ignorePlacement: paint['text-ignore-placement'],
+            optional: paint['text-optional'],
+            visibility: paint.visibility,
+            opacity: paint['text-opacity'],
+            color: paint['text-color'],
+            haloColor: paint['text-halo-color'],
+            haloWidth: paint['text-halo-width'],
+            haloBlur: paint['text-halo-blur'],
+            translate: paint['text-translate'],
+            translateAnchor: paint['text-translate-anchor']
+        };
+    }
+
+    /**
+     * Creates a GeoStylerStyle-FillSymbolizer from a Mapbox Style Layer.
+     *
+     * @param {any} layer A Mapbox Style Layer
+     * @return {FillSymbolizer} A GeoStylerStyle-FillSymbolizer
+     */
+    getFillSymbolizerFromMapboxLayer(paint: any): FillSymbolizer {
+        return {
+            kind: 'Fill',
+            visibility: paint.visibility,
```

Comment:
See above.

Reply:
Solved in https://github.com/terrestris/geostyler-mapbox-parser/pull/1/commits/b8912d60e00672bff2a78e67b869744c5cb9f8b6